### PR TITLE
Add a bio for @tallstuff1964

### PR DESCRIPTION
**Pull request by tallstuff1964**
This Pull Request adds a bio file for @tallstuff1964 as requested in #16 .

Hey, @jgjucha, I would :heart: if you had some :clock: to put some :eyes: on this and let me know your :thought_balloon:s.

### CODE REVIEW

**Comment by tallstuff1964** [comment]
These changes look good.

File: docs/AngelaIveyBio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Angela Ivey Bio
+- Name: Angela Ivey
+- Email:  angela.c.ivey@uscis.dhs.gov
+- Occupation: Task Lead
+- Location: Rosslyn, VA
```
Comment (tallstuff1964) [inline docs/AngelaIveyBio.md:5]: Where on earth is Rosslyn?

Comment (tallstuff1964) [inline docs/AngelaIveyBio.md:5]: Does Rosslyn have two "r"'s

Comment (pprabhala) [inline docs/AngelaIveyBio.md:5]: Rosslyn, Va is in Virginia :smile:

Comment (tallstuff1964) [inline docs/AngelaIveyBio.md:5]: Line 5 is correct.  No corrections are necessary

Comment (gvarma) [inline docs/AngelaIveyBio.md:5]: Is Rosslyn is close to DC?

**Comment by tallstuff1964** [comment]
This pull request has been reviewed.

**Pull request merged**